### DISCUSSION TITLE
Is there any discussion on fully implementing fluent assertions in xUnit?

### DISCUSSION THREAD

**Discussion by PhilParisot** [xunit/xunit#2631] [Feedback Request]
From an enterprise perspective, getting fluent assertions approved through management and all the red tape is time and energy consuming, not the end of the world by any means but a little painful.

With fluent assertions quickly becoming an industry standard, are there any plans on making it a hard dependency?

It could've avoided the duplication of work for `Assert.Equivalent()` for example.

If this discussion has already happened I apologize in advance I couldn't find anything with a quick search.

Food for thought.

**Comment by bartelink** [comment]
Maintaining a fluent assertions lib is lots of work. Some people believe it also produces a lot of value. I personally happen to use a custom assertions library, but appreciate how bullet proof xUnit is for doing everything else. I also appreciate its basic assertions for where people are not interested in a flowery assertions lib.

So, I'd say:
- xUnit is not in the fluent assertions game, it has a single terse consistent assertions lib. It has some minor warts by virtue of being a >10yo framework that has not had ever had much junk or redundancy (and hence obsoleted)
- there's nothing blocking others adding one
- so why wouldnt anyone change anything?
